### PR TITLE
Adds SSE use case scenario

### PR DESCRIPTION
**Pull request by waldemarnt**
This use case scenario explain how to subscribe to an TrackingData event stream using
SSE on Telemetry API.
After:
screen shot 2016-08-08 at 4 35 02 pm

### CODE REVIEW

**Comment by jeffhsta** [comment]
:+1:

File: source/includes/use_case_scenarios/_streaming_data_telemetry_api.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+             "trackingPoint":"fe4c0024-600c-4757-b25c-2f39ded45ec6",
+             "canVariable":"7ed4b4c4-76e5-4dfb-b548-ea35b9f50276"
+           }
+       }
+```
+
+Every time that an [Equipment](#equipment) send informations like position, status and [CAN variables](#can-variables) the `Telemetry API` separates
+this information into `TrackingPoint` and `TrackingData`.
+
+[TrackingPoint](#tracking-points) is the point where the information was collected by the machine. Each `TrackingPoint` has one
```
Comment (carolinaborim) [inline source/includes/use_case_scenarios/_streaming_data_telemetry_api.md]: [TrackingPoint](#tracking-points) is where the information was collected by the machine.

File: source/includes/use_case_scenarios/_streaming_data_telemetry_api.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+[Duty](#duty) that is related to the `state of the machine` in the point.
```
Comment (carolinaborim) [inline source/includes/use_case_scenarios/_streaming_data_telemetry_api.md]: [Duty](#duty) related to the `state of the machine` in the point.

File: source/includes/use_case_scenarios/_streaming_data_telemetry_api.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+[TrackingData](#tracking-data) is the data sent by an `Equipment` in a point. One `TrackingData` is created for each `CAN variable` value received
```
Comment (carolinaborim) [inline source/includes/use_case_scenarios/_streaming_data_telemetry_api.md]: [TrackingData](#tracking-data) is sent by an `Equipment` in a point.

**Comment by carolinaborim** [comment]
👍

**Pull request merged**